db/migrate/20250114144917_create_mentor_call_off_contracts.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CreateMentorCallOffContracts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :mentor_call_off_contracts do |t|
+      t.belongs_to :lead_provider, null: false, foreign_key: true, type: :uuid, index: true
+      t.belongs_to :cohort, null: false, foreign_key: true, type: :uuid, index: true
+
+      t.string :version, null: false, default: "0.0.1"
```

Comment:
Are we just sticking to the call off contract versioning/logic?

Reply:
Think so.. but also we need to sync up this value with `contract_version` from statements.

Reply:
As discussed on call yesterday we'll keep the current version implementation